compiler/src/dotty/tools/dotc/core/Denotations.scala

```diff
@@ -298,14 +298,13 @@ object Denotations {
                        name: Name,
                        site: Denotation = NoDenotation,
                        args: List[Type] = Nil,
-                       source: AbstractFile | Null = null,
```

Comment:
shouldn't this also be replaced by CompilationUnitInfo? it seems the stub symbol reports what it is given

Reply:
ok I see in the source code it is never called with a source argument